README.md

```diff
@@ -173,4 +89,4 @@ The goal for Mentat is to become the best way to write code. Some big improvemen
 
 If this is something you'd like to work on, jump right in! And if you want to join the team to work on this full time, message [@bio_bootloader](https://twitter.com/bio_bootloader) on twitter.
 
-To find something specific to work on, take a look at [open issues](https://github.com/AbanteAI/mentat/issues) and/or check this Github Project: [Mentat Development](https://github.com/users/biobootloader/projects/2)
+To find something specific to work on, take a look at [open issues](https://github.com/AbanteAI/mentat/issues).
```

Comment:
It doesn't seem like we really use the project so I thought it was better to remove the link.